controls/sysctl_spec.rb

```diff
@@ -22,12 +22,14 @@
   impact 1.0
   title 'IPv4 Forwarding'
   desc "If you're not intending for your system to forward traffic between interfaces, or if you only have a single interface, the forwarding function must be disable."
-  describe kernel_parameter('net.ipv4.ip_forward') do
-    its(:value) { should eq 0 }
-  end
-  describe kernel_parameter('net.ipv4.conf.all.forwarding') do
-    its(:value) { should eq 0 }
-  end
+#  unless defined? ENV['sysctl_forwarding']
+    describe kernel_parameter('net.ipv4.ip_forward') do
+      its(:value) { should eq 0 }
+    end
+    describe kernel_parameter('net.ipv4.conf.all.forwarding') do
+      its(:value) { should eq 0 }
+    end
+#  end
```

Comment:
we can add a only_if condition to the control block for servers where forwarding is required:
```
control 'sysctl-01' do
    impact 1.0
    title 'IPv4 Forwarding'
    desc "If you're not intending for your system to forward traffic between interfaces, or if you only have a single interface, the forwarding function must be disable."
    describe kernel_parameter('net.ipv4.ip_forward') do
      its(:value) { should eq 0 }
    end
    describe kernel_parameter('net.ipv4.conf.all.forwarding') do
      its(:value) { should eq 0 }
    end
    only_if { sysctl_forwarding == true }
end
```